package.json

```diff
@@ -8,7 +8,7 @@
     "skyux": "bin/skyux.js"
   },
   "scripts": {
-    "coverage": "istanbul cover --report=html jasmine JASMINE_CONFIG_PATH=jasmine.json",
+    "coverage": "istanbul cover --report=cobertura --report=html node_modules/jasmine/bin/jasmine.js -- JASMINE_CONFIG_PATH=jasmine.json",
```

Comment:
Leaving this as `jasmine` causes `istanbul` to run `node_modules\.bin\jasmine.CMD`. See https://github.com/gotwarlost/istanbul/issues/13#issuecomment-32001190